src/pylexibank/commands/consonant_clusters.py

```diff
@@ -28,10 +28,12 @@ def run(args):
 
 
 def compute_consonant_cluster(word, sound_names):
-    out = [[]] if sound_names[0].split(" ")[-1] in ["consonant", "cluster"] else []
+    out = [[]] if sound_names[0].split(" ")[-1] in ["consonant", "cluster", "∼"] else []
     for i, sound in enumerate(sound_names):
         if sound.split(" ")[-1] in ["diphthong", "vowel", "tone", "�", "marker"]:
             out += [[]]
+        elif sound.split(" ")[0] == 'syllabic':
```

Comment:
Would it be saver to turn this into `"syllabic" in sound.split(" ")` ? I do not know by hard if the order of names would always place syllabic in front...

Reply:
CLTS does put 'syllabic' at the beginning for consonants, so the current implementation is safe